exporters/zipkin/src/recordable.cc

```diff
@@ -215,7 +224,18 @@ void Recordable::SetDuration(std::chrono::nanoseconds duration) noexcept
   span_["duration"] = duration.count();
 }
 
-void Recordable::SetSpanKind(opentelemetry::trace::SpanKind span_kind) noexcept {}
+void Recordable::SetSpanKind(opentelemetry::trace::SpanKind span_kind) noexcept
+{
+  auto span_iter = kSpanKindMap.find(span_kind);
+  if (span_iter != kSpanKindMap.end())
+  {
+    span_["kind"] = span_iter->second;
+  }
+  else
+  {
```

Comment:
I'm just wondering if you really need the `nullptr` assignment here, since the set is complete in the map... Also I think if you try to access the `span_["kind"]` property on JSON object elsewhere, it should already return you `nullptr` or an empty string anyways.... The only problem here might be is if there's a new type of `SpanKind` added to API, but not added to the map above.

Reply:
Good point. Will remove it.

Reply:
Sounds good.